src/Filesystem.cpp

```diff
@@ -250,8 +196,10 @@ File Filesystem::open(const std::string& filename) const
  */
 bool Filesystem::makeDirectory(const std::string& path) const
 {
-	if (!PHYSFS_isInit()) { throw filesystem_not_initialized(); }
-	return PHYSFS_mkdir(path.c_str()) != 0;
+    namespace FS = std::filesystem;
+    if(this->isInit()) { throw filesystem_not_initialized(); }
```

Comment:
This appears to be incorrect -- is the intention to check that the FS is initialized and if not to throw?

Additionally, why the explicit 'this' pointer?

Reply:
The explicit 'this' pointer was me fighting with intellisense and I forgot to remove it. :P